tests/sites_tests/tests.py

```diff
@@ -205,6 +206,12 @@ def test_site_natural_key(self):
         self.assertEqual(Site.objects.get_by_natural_key(self.site.domain), self.site)
         self.assertEqual(self.site.natural_key(), (self.site.domain,))
 
+    @override_settings(SITE_ID='1')
+    def test_check_site_id(self):
+        error = checks.run_checks()
+        expected = [checks.Error(msg='SITE_ID must be an integer', obj='sites.E101', )]
+        self.assertEqual(expected, error)
```

Comment:
It looks like this test is failing when run alongside other test apps with invalid models. I think you can change this assertion to check for the presence of at least the expected error:
```suggestion
        errors = checks.run_checks()
        expected = checks.Error(msg='SITE_ID must be an integer', obj='sites.E101')
        self.assertIn(expected, errors)
```